src/Symfony/Component/PropertyAccess/NamingStrategy/NamingStrategyInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\NamingStrategy;
+
+/**
+ * @author David Badura <d.a.badura@gmail.com>
+ */
+interface NamingStrategyInterface
+{
+    /**
+     * returns all possible getters.
```

Comment:
capital first. Im not sure about the duplication in terms of typehints in docblocks on interfaces. Feels useless also :)